state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -992,7 +993,11 @@ pub mod pallet {
 				// NOTE: DCA not yet supported for swaps from the contract
 				None,
 				SwapOrigin::Vault { tx_hash },
-			)?;
+			)
+			.is_err()
+			{
+				log_or_panic!("Regular swap request should never fail");
```

Comment:
CCM is the only case that the function can fail in

Reply:
Updated the comments

Reply:
Not really sure what you mean here... it looks like you removed this check and "log_or_panic" which I think is correct, but doesn't match what you're saying here... "Updated the comments. Also changed to log::error so it is testable." just want to double check

Reply:
Oh Sorry! i got confused while working on 2 separate PRs. 

This PR here:
https://github.com/chainflip-io/chainflip-backend/pull/5242
removed the `log_or_panic` to log::error, so this code path is testable.